aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/UpdateHandler.java

```diff
@@ -112,15 +117,33 @@ public UpdateHandler(TagHelper tagHelper) {
             logger.log(String.format("INFO Updated tags for patch baseline %s %n", baselineId));
 
             // Set to default patch baseline
-            if (BooleanUtils.isTrue(model.getDefaultBaseline()) && !BooleanUtils.isTrue(previousModel.getDefaultBaseline())){
+           // if (BooleanUtils.isTrue(model.getDefaultBaseline()) && !BooleanUtils.isTrue(previousModel.getDefaultBaseline())){
+            if(model.getDefaultBaseline()!=null && BooleanUtils.isTrue(model.getDefaultBaseline()) ){
+
                 RegisterDefaultPatchBaselineRequest registerDefaultPatchBaselineRequest = RegisterDefaultPatchBaselineRequest.builder()
                         .baselineId(baselineId)
                         .build();
                 RegisterDefaultPatchBaselineResponse registerDefaultPatchBaselineResponse =
                         proxy.injectCredentialsAndInvokeV2(registerDefaultPatchBaselineRequest, ssmClient::registerDefaultPatchBaseline);
 
-                logger.log(String.format("INFO Registered patch baseline %s to default patch baseline successfully %n", baselineId));
+                if(registerDefaultPatchBaselineResponse== null){
+
+                    return ProgressEvent.<ResourceModel, CallbackContext>builder()
+                            .resourceModel(model)
+                            .status(OperationStatus.SUCCESS)
+                            .build();
+
+                }else{
+                    logger.log(String.format("INFO Registered patch baseline %s to default patch baseline successfully %n", baselineId));
+                }
             }
+           /* if(model.getDefaultBaseline().booleanValue()) {
```

Comment:
please remove those comments if they are not meant to be used

Reply:
Removed the unused comment code changes from the file